### PR TITLE
Toward working

### PR DESCRIPTION
**Pull request by humphd**
This is a start at getting things to work.  We can't touch `ProjectManager` until it's all done loading the project, so we can't start the server early.  This fixes that.  We also need to open the browser only after the project is open, or it destroys the second pane that holds the iframe.

I've also fixed a few stray typos, incorrect method calls, etc.  We still have the issue that this is using `setLauncher()` which isn't in Brackets yet.  We either need to merge that into our bramble branch, or somehow do without it.

@sedge, I'll let you run with this, but I did it as a PR in case you wanted to see what's up.

### CODE REVIEW

File: main.js [hunk 1]
```diff
@@ -20,54 +20,50 @@ define(function (require, exports, module) {
...
-        // a file exists to be opened as a project. Once it's opened, we can
-        // start the live preview.
-        function _beginLivePreview() {
-            LiveDevelopment.open();
-        }
-        ProjectManager.one("projectOpen", _beginLivePreview);
+        LiveDevelopment.open();
     }
-    LiveDevelopment.one("statusChange", _configureLiveDev);
+    ProjectManager.one("projectOpen", _configureLiveDev);
```
Comment (sedge) [inline main.js:58]: Nice one on this.

**Comment by sedge** [comment]
This all looks good to me. I'll test your patch tomorrow and go from there.

**Comment by humphd** [comment]
I did a bit of digging this morning, but I'm out of time.  What I found is that our call to `setLauncher()` with our custom Launcher code is happening too soon, and it gets overwritten by the default `setLauncher()` call here https://github.com/adobe/brackets/pull/10558/files#diff-e31c028c519c0302491877c1d9c0bad1R840.  Probably we need to deal with the fact that this is being set in `AppInit.appReady` which is really late in the startup.

**Comment by sedge** [comment]
I left this in a state where I'm pretty sure the project is opened, the transport and launcher are successfully swapped out but the `Launcher.launch(url)` call crashes. I'll keep digging, and the rest of the team has a parallel patch based on your work.

File: main.js [hunk 2]
```diff
@@ -20,16 +20,44 @@ define(function (require, exports, module) {
...
+    // We wait until the LiveDevelopment module is initialized and the project loaded
+    // so we can safely swap our transport and launcher modules for
+    // the defaults and start LiveDev.
+    function _configureLiveDev() {
+        // Turn preview iFrame On
+        Browser.init();
+
+        function _configureModules() {
+            // Set up our transport and plug it into live-dev
+            PostMessageTransport.setIframe(Browser.getBrowserIframe());
```
Comment (AJDBenner) [inline main.js:44]: getBrowserIframe should be getBrowserIFrame

Comment (humphd) [inline main.js:44]: Actually, let's use `*Iframe` everywhere, to match what Brackets code does (i.e., capital `I` and lowercase `f`)

Comment (sedge) [inline main.js:44]: @KeniMorri This means you need to update the `iframe-browser.js` file to match.

**Pull request merged**